pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/plan/server/ServerPlanRequestUtils.java

```diff
@@ -106,7 +106,7 @@ public static OpChain compileLeafStage(OpChainExecutionContext executionContext,
       BiConsumer<PlanNode, MultiStageOperator> relationConsumer,
       boolean explain) {
     long queryArrivalTimeMs = System.currentTimeMillis();
-    MdcExecutor mdcExecutor = new MdcExecutor(executorService) {
+    ExecutorService decoratedExecutor = new MdcExecutor(executorService) {
```

Comment:
Is there any reason to apply this change? Maybe you needed that before but now that you are creating the executor in another place it is not needed?

Reply:
You guessed it... it made sense before, but no longer does. Undone.

a974159d74a63116ef234b0617ca60aaac5b042f